### DISCUSSION TITLE
下雨天就別外出，Amazon 特價入手 Chromecast 看 Disney+

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#447] [General]
下雨天就別外出，Amazon 特價入手 Chromecast 看 Disney+
不過買家庭方案還是最便宜啦
https://tw.news.yahoo.com/%E4%B8%8B%E9%9B%A8%E5%A4%A9%E5%B0%B1%E5%88%A5%E5%A4%96%E5%87%BA-amazon-%E7%89%B9%E5%83%B9%E5%85%A5%E6%89%8B-chromecast-%E7%9C%8B-140042867.html

**Comment by number32984** [comment]
不用出門就是讚啦

**Comment by brian431** [comment]
我也有買個Chromecast 還蠻好用的。

**Comment by Aerofe** [comment]
ChromeCast之前看別人介紹過，感覺挺不錯的，之後可能也會買一個來用。

**Comment by BingRong1217** [comment]
很棒的推薦！

**Comment by Yilin20021016** [comment]
感覺這個很合我不出門的人 不論有沒有下雨

**Comment by HsuChen88** [comment]
我必須停止成為寄生仔，自立自強。
